hashicorp/waypoint
# ui: add missing `--badge-text` for light mode

## Why the change?

Extracted from #1668

While working on #1668 I noticed this variable definition was missing from the light-mode side of `variables.scss`.

This sneaks in another change from the #1668 designs, making the light-mode badge background a light blue rather than gray.

## What does it look like?

| Before | After |
| --- | --- |
| <img alt="light-mode before" src="https://user-images.githubusercontent.com/34030/124585819-fc374880-de55-11eb-95ca-e17af4a3ca0c.png"> | <img alt="light-mode after" src="https://user-images.githubusercontent.com/34030/124585826-fd687580-de55-11eb-8b53-038a2980a36b.png"> |
| <img alt="dark-mode before" src="https://user-images.githubusercontent.com/34030/124585829-fe010c00-de55-11eb-9f7c-f3223b6212af.png"> | <img alt="dark-mode after" src="https://user-images.githubusercontent.com/34030/124585832-fe99a280-de55-11eb-8edf-1c091eb2a57d.png"> |

## How do I test it?

1. Check out the branch
2. Run the ui dev server
   ```
   cd ui
   yarn start
   ```
3. Visit [http://localhost:4200](http://localhost:4200)
4. Verify it all looks good
